p/themes/base-theme/frss.css

```diff
@@ -1206,14 +1206,15 @@ a.website:hover .favicon {
 }
 
 .flux .item.title .summary {
-	max-height: 3em;
+	display: -webkit-box;
```

Comment:
I am not so found of introducing proprietary attributes, if that could be avoided. What about non-Webkit browsers?

Reply:
Unfortunately not. But it is implemented in the major browsers:
https://caniuse.com/?search=-webkit-box